environmental-impact.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Environmental impact
+---
+Let's face the truth: Conferences like LambdAle are not very environmental friendly. Tens or hundreds of people meeting for an event that typically lasts no more than a day or two. Some live close-by and come by public transport or car, but many fly in from other countries of the continent, or even travel from overseas. It is especially these flights that account for a lot of CO2 emissions. We don't believe in flat earth, but we do believe in global warming and that flying is a major contributor to it. So let's call a spade a spade - conferences are an environmental disaster.
```

Comment:
> many fly in from other countries of the continent, or even travel from overseas

This phrasing sounds a little unnatural. How about “many fly from another country or even another continent”?

Reply:
Done